src/plugins/data/public/ui/typeahead/suggestions_component.tsx

```diff
@@ -29,12 +29,45 @@ interface Props {
   show: boolean;
   suggestions: QuerySuggestion[];
   loadMore: () => void;
+  queryBarRect?: DOMRect;
+  dropdownHeight?: string;
 }
 
 export class SuggestionsComponent extends Component<Props> {
   private childNodes: HTMLDivElement[] = [];
   private parentNode: HTMLDivElement | null = null;
 
+  kbnTypeaheadDivRefInstance: RefObject<HTMLDivElement> = createRef();
+
+  updatePosition = () => {
+    const kbnTypeaheadDiv = this.kbnTypeaheadDivRefInstance.current;
+    const { queryBarRect } = this.props;
+
+    if (queryBarRect && kbnTypeaheadDiv) {
+      const documentHeight = document.documentElement.clientHeight || window.innerHeight;
+      const suggestionsListOffsetBottom = 20;
```

Comment:
nit: we could gather all the constants on a constants file. It will be cleaner I think and easier to find. As far as I can understand these are `suggestionsListOffsetBottom `, `defaultMaxHeight ` and `dropdownHeight `

Reply:
+1